tests/acceptance/features/cliDataExporter/exportFileWithEmptyTrashBinDataExporterIssue209.feature

```diff
@@ -0,0 +1,12 @@
+@cli
+Feature: An administrator wants to export the files of his user using the commandline
```

Comment:
DON'T we follow this rule for core app? 
```
As an admin
I want to .............
So that ...................
```

Reply:
since this is a bug demonstration feature file I don't think it's necessary here